src/app/dashboard/Missions/MissionFundedStatusRow.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+import { Mission } from "../../model";
+import AttachMoneyIcon from "@material-ui/icons/AttachMoney";
+import { Box, Container, Grid, Paper } from "@material-ui/core";
+
+const Row = ({ children, Icon }) => {
+  if (!children) return null;
+  return (
+    <Grid container wrap="nowrap" alignItems="center">
+      <Box marginRight="5px" width="20px">
+        {Icon && <Icon color="primary" />}
+      </Box>
+      {children}
+    </Grid>
+  );
+};
+
+const MissionFundedStatusRow = ({ classes, mission }) => {
+  let missionFundedStatusText;
+  let fundedDateText = mission.fundedDate ? " on " + mission.fundedDate : "";
+  switch (mission?.fundedStatus) {
+    case Mission.FundedStatus.fundedbydonation:
+      missionFundedStatusText = "Funded By Donation" + fundedDateText;
+      break;
+    case Mission.FundedStatus.fundedbyrecipient:
+      missionFundedStatusText = "Funded By Recipient" + fundedDateText;
+      break;
+    case Mission.FundedStatus.fundingnotneeded:
+      missionFundedStatusText = "Funding Not Needed";
+      break;
+    case Mission.FundedStatus.notfunded:
+      missionFundedStatusText = "Not Yet Funded";
+      break;
```

Comment:
Can we use template literals in stead of + for string concatenations?

If `fundedDateText` isn't provided properly, you're gonna see `Funded By Donation12/12/20` or whatever the date text looks like

`Funded by donation: ${fundedDateText}` probably be a little cleaner.

Reply:
Done!